pyproject.toml

```diff
@@ -42,6 +42,7 @@ dependencies = [
     "markdownify >= 0.11",
     "deprecation >= 2.1",
     "packaging",
+    "pandas >= 2.1.4"
```

Comment:
pandas is a test dependency or library dependency?

Reply:
> Where did the file `examples/somersault_01.01.02.pdx` come from?

The file was derived from the file `somersault.pdx` by adapting `somersaultecu.py`

> pandas is a test dependency or library dependency?

pandas was used by several functions but has been replaced

Reply:
what are the adaptations?

@andlaus are you ok with adding a binary file without the logic that created it?

Reply:
- new ecu variant `somersault_young`
- deleted SingleEcuJob `compulsory_program`
- deleted DiagService `set_operation_params`
- new DiagService `flicflac`
- renamed DiagService `start_session` from `session_start` to `start_session`
- changed pos. response parameter of `session_start`
(DOP of `can_do_backward_flips` from `uint8` to `boolean`)
- changed pos. response parameter of `tester_present`
(value of `status` from `0x00` to `0x0043` and type from `uint8` to `uint16`)
- changed pos. response parameter of `grudging_backward`
(byte position of `num_flips_done` from `None` to `1`)
(param type of `grumpiness_level` from `ValueParameter` to `CodedConstParameter`, 
incl. byte position from `None` to `2`, value = `0x05`)
- changed pos. response parameter of `status_report`
(byte position of `happiness_level` from `2` to `1`)
(byte position of `dizzyness_level` from `1` to `2`)
- changed request parameter of `stop_session`
(type of `id` from `uint8` to `uint16`)
- changed request parameter of `tester_present`
(value of `id` from `0x0` to `0x01`)
- changed request parameter of `do_backward_flips`
(short_name of `backward_check` from `backward_soberness_check` to `backward_check`)

![image](https://github.com/mercedes-benz/odxtools/assets/142314987/e399d073-85f7-4d9b-92bc-ef3b9451af5b)

Reply:
> @andlaus are you ok with adding a binary file without the logic that created it?

I suppose in this case it is okay, because that new file is basically a showcase for the `compare` tool. (I'd prefer to have a script doing these modifications in-repo, though.)

can this file be renamed to something which better conveys the fact that it is a modified file of the original? (something like `somersault_modified.pdx`.)